release/tar/linux/opensearch-tar-build.sh

```diff
@@ -105,7 +113,7 @@ LIBS=`yq eval '.products.opensearch.libs' $MANIFEST_FILE | sed s/^-// | sed -e '
 
 # # Download plugins to local 
 mkdir $WORKING_DIR
-mkdir $TARGET_DIR
+mkdir -p $TARGET_DIR
```

Comment:
Why does TARGET_DIR have -p while WORKING_DIR does not?

Reply:
I mentioned the use case in PR description